otelib/exceptions.py

```diff
@@ -5,6 +5,10 @@ class BaseOtelibException(Exception):
     """A base OTElib exception."""
 
 
+class AuthorizationError(BaseOtelibException):
```

Comment:
Are you using this exception anywhere? If not, I don't see a reason to add it.

Reply:
I originally thought to use it in case if the response error code is an 401. But it did not resolve in using it, hence I removed it again.